product-web-app/views/view-single-jobRole.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Can you create template for HTML/Header document
See: https://mozilla.github.io/nunjucks/templating.html